metadata-ingestion/src/datahub/ingestion/source/unity/config.py

```diff
@@ -50,6 +50,11 @@ class UnityCatalogSourceConfig(StatefulIngestionConfigBase, DatasetSourceConfigM
         description="Option to enable/disable lineage generation.",
     )
 
+    include_table_ownership: bool = pydantic.Field(
+        default=True,
```

Comment:
I'd recommend that we default this to false (we should be in other connectors as well).

This is because you can easily mint ghost users when this is done